Rewrite IT case to remove Vertx web client

genryxy opened the pull request:
Closes #150 
`HelmSliceIT` was refactored to remove `vertx-web-client` dependency as the same could be done using

olenagerasimova commented on src/test/java/com/artipie/helm/HelmSliceIT.java:
```diff
@@ -24,143 +24,60 @@
-        }
-    }
-
+final class HelmSliceIT {
```
@genryxy hm, now this test is not integration test as it does not use/communicate with any external systems...

olenagerasimova reviewed (changes requested):
@genryxy please, consider my comment

genryxy commented:
@olenagerasimova thanks, corrected. Moved test case from `HelmITCase` to `HelmCompatibilityITCase` and renamed `HelmCompatibilityITCase` to `HelmITCase`

olenagerasimova commented:
@genryxy thanks
